src/MicroDB/Database.php

```diff
@@ -2,38 +2,55 @@
 
 namespace MicroDB;
 
+use Exception;
+
 /**
  * A file-based JSON object database
  */
-class Database {
+class Database
+{
```

Comment:
Sorry, but according to the PSR-1 standards the { should be on the next line.
http://www.php-fig.org/psr/psr-1/